calico-cloud_versioned_docs/version-3.16/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -1,62 +1,97 @@
 ---
-description: Identify vulnerabilities in Kubernetes clusters.
+description: Detect vulnerabilities in a Kubernetes cluster.
 ---
 
-# Enable automatic image scanning inside a cluster
+# Scan all images in a Kubernetes cluster
 
-## Big picture
+Scan all images in a Kubernetes cluster for continuous clean bill of health and defense in depth.
 
-Scan Kubernetes workloads automatically using {{prodname}} Image Assurance cluster scanner.
+Common use cases for scanning in a Kubernetes cluster are: 
 
+- Images may pass scanning during the build phase, but they could contain vulnerabilities days or weeks later
+- Third-party images pulled from public registries are often not scanned in build pipelines and can contain Critical or High vulnerabilities
+- Application teams often build one-off images outside their pipeline to make an emergency patch and fix a critical bug
 
-## Before you begin
+## Image Assurance cluster scanner
 
-**Cluster requirements**
+The cluster scanner is out-of-the-box ready to enable without configuration. Running images are checked for new vulnerabilities every 24 hours, and scan results are reported back to {{prodname}} Manager UI in the Image dashboard.
 
-To be included in scan results, cluster must be:
+The cluster scanner runs as a daemonset in your managed clusters where images are located. You can enable the feature per managed cluster, but it must be enabled on all nodes if you want to scan all images in your cluster. 
```

Comment:
This seems to imply that the user has to do something extra (on top of enabling the feature for a cluster) in order to get it running on all nodes. 

That is not the case. If a user enables for a cluster, then it automatically installs on all nodes (by way of the daemon set).

Reply:
Fixed.